code/_globalvars/lists/devil.dm

```diff
@@ -0,0 +1,8 @@
+//what could possibly go wrong
+var/list/devil_machines = typecacheof(/obj/item/circuitboard, TRUE)  - list(
```

Comment:
This file is the skeleton of a machine whitelisting system where a person could sell their soul for a machine (and parts to build it) of their choice. I've not gotten around to fully fleshing that out, however.